data/Canada/Commons/sources/reconciliation/twitter.csv

```diff
@@ -0,0 +1,252 @@
+id,uuid
```

Comment:
what is this file? I can't see how it's used (`twitter-lists.csv` seems to be the actual reconciliation file.) Is this left behind from an earlier attempt?